pkg/odo/cli/component/link.go

```diff
@@ -63,6 +65,23 @@ odo link dh-postgresql-apb
 Now backend has 2 ENV variables it can use:
 DB_USER=luke
 DB_PASSWORD=secret`
+
+	linkExampleExperimental = ktemplates.Examples(`# Link the current component to the 'EtcdCluster' named 'example'
```

Comment:
it is getting printed as 
```
Examples:
  # Link the current component to the 'EtcdCluster' named 'example'
  odo link link EtcdCluster/example
```
remove the double link

Reply:
Thanks! Fixed.